guildwars2/guild/general.py

```diff
@@ -17,23 +17,31 @@ class GeneralGuild:
 
     @guild.command(name="info")
     @commands.cooldown(1, 20, BucketType.user)
-    async def guild_info(self, ctx, *, guild_name: str):
+    async def guild_info(self, ctx, *, guild_name=None):
         """General guild stats
 
         Required permissions: guilds
         """
+        # Read preferred guild from DB
+        guild_id = await self.get_preferred_guild(ctx)
+        # Get Guild name if ID already stored
+        if guild_id:
+            guild_name = await self.guildid_to_guildname(ctx, guild_id)
+        elif guild_name is not None:
+            guild_id = await self.guildname_to_guildid(ctx, guild_name)
+        else:
+            if not ctx.guild:
+                await ctx.send("Preferred guild is only available on server.")
```

Comment:
Probably should just skip out this message and only send help, as if an argument was missing

Reply:
Removed the message for all guild commands